src/components/BuildBaronAndAnnotations/BBFIleTicket.tsx

```diff
@@ -79,13 +78,15 @@ export const FileTicket: React.FC<FileTicketProps> = ({
       okButtonProps={{ loading: loadingFileJiraTicket }}
       cancelButtonProps={{ disabled: loadingFileJiraTicket }}
     >
-      <Button
-        data-cy="file-ticket-button"
-        variant={Variant.Primary}
-        size="xsmall"
-      >
-        {buttonText}
-      </Button>
+      <ButtonWrapper>
+        <Button
```

Comment:
You can alternatively just apply the styles from ButtonWrapper directly to the button if you dont need it to be wrapped in a div.

Reply:
Ah, I used a wrapper because otherwise the button was showing up directly next to the thing above it; Chaya mentioned that a div would fix that.